CHANGELOG-1.8.md

```diff
@@ -416,7 +416,7 @@ Consider the following changes, limitations, and guidelines before you upgrade:
 
   * The `--audit-policy-file` option is required if the `AdvancedAudit` feature is not explicitly turned off (`--feature-gates=AdvancedAudit=false`) on the API server.
   * The audit log file defaults to JSON encoding when using the advanced auditing feature gate.
-  * The `--audit-policy-file` option requires `kind` and `apiVersion` fields specifying what format version the `Policy` is using.
+  * An audit policy file without an `apiVersion` or `kind` field may be treated as invalid.
```

Comment:
why not `is treated as invalid`?

Reply:
https://github.com/kubernetes/kubernetes/issues/54254#issuecomment-338070650

Reply:
I don't understand @liggitt's comment or its context. This PR here treats those files as invalid. Why soft language in the changelog?

Reply:
In 1.8.0 1.8.1, such policy files are not treated as invalid.

Reply:
Do we cherry pick this to 1.8?
@sttts

Reply:
Got it. Do we actually reference this file instead of the 1.8 branch one?

Reply:
> Do we cherry pick this to 1.8?

We should IMO.

Reply:
> Do we actually reference this file instead of the 1.8 branch one?

I don't know.

Reply:
we don't cherry pick CHANGELOG changes to release branches